src/utils/temp-dir.ts

```diff
@@ -0,0 +1,22 @@
+import os from "os";
+// @ts-ignore
+import fs from "fs";
+// @ts-ignore
+import path from "path";
+import { v4 as uuidv4 } from 'uuid';
+
+import { logger } from './logger';
+
+export function createTempDirectory() {
+  // @ts-ignore
```

Comment:
Why are all these `@ts-ignore` annotations present? 🤔

Reply:
I had some problems with `ts-node` which always complains that I have duplicated core packages in source code, but after bumping TS version it seems that fixed it.